src/main/java/sg/ncl/MainController.java

```diff
@@ -4366,7 +4404,7 @@ private String getScenarioContentsFromFile(String scenarioFileName) throws WebSe
                 sb.append(line);
                 sb.append(System.getProperty("line.separator"));
             }
-            log.info("Experiment ns file contents: {}", sb);
+            //log.info("Experiment ns file contents: {}", sb);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)